### ISSUE TITLE
Enable default config transitions in e2e tests

### ISSUE THREAD

**Issue by dianabarsan** [medic/medic#6026]
**Describe the issue**
The default config was updated to enable two transitions: `generate_patient_id_on_people` and `death_reporting`. 
We also recently started using the default config when running our e2e tests suite. Having _any_ enabled transitions (especially `generate_patient_id_on_people`  which updates every contact doc uploaded) can cause our e2e tests to fail in unexpected ways. 
To release `3.7`, the quick solution is to remove transitions from the config before uploading when running e2e tests. 

**Describe the improvement you'd like**
We should update the existent tests setup or assertions so they execute successfully using the default config.

**Comment by garethbowen** [comment]
Pushing back to 3.9.0 as there's no urgency on this one.

**Comment by garethbowen** [comment]
Moved to 3.12.0 to free up engineers to work on 3.11.0.

**Comment by dianabarsan** [comment]
Yes, we still disable transitions when running e2e tests: https://github.com/medic/cht-core/blob/master/tests/utils.js#L637
